api/api.proto

```diff
@@ -476,6 +520,18 @@ message CapacityUsageInfo {
   int64 total_bytes = 3;
 }
 
+// A SdkStoragePolicy represents minimum set of volume specs to be 
+// follow while creating volumes.
+// If enforce in OpenStoragePolicy service, VolumeSpecPolicy will be 
+// used before creating volume to ensure minimum volume creation rules
+// are enforced.
+message SdkStoragePolicy {
+  // Name of storage policy.
+  string name = 1;
+  // VolumeSpecs to apply while creating volume.
+  VolumeSpecPolicy policy = 2;
```

Comment:
@lpabon , should we also include **IsEnforced** here. So that user with enumerate or inspect call will know policy is enforced or not. That way they don't need to make another call for getting enforcement details ?

Reply:
I think you have a great idea, but if we do it with IsEnforced, they would have to enumerate and search each. Instead, let's make a new API called GetEnforced (which is the one your already have implemented). That way, they can call that to verify which one is enforced.